hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/FeeManager.java

```diff
@@ -193,7 +206,12 @@ public FeeCalculator createFeeCalculator(
         final var feeData = getFeeData(functionality, consensusTime, SubType.DEFAULT);
 
         // Create the fee calculator
-        return new FeeCalculatorImpl(feeData, exchangeRateManager.activeRate(consensusTime));
+        return new FeeCalculatorImpl(
+                feeData,
+                exchangeRateManager.activeRate(consensusTime),
+                congestionMultipliers,
+                this.state,
```

Comment:
I am pretty sure this is not correct. If the state is required, we probably need to add it as a parameter to `createFeeCalculator()`.

Reply:
The sate is removed from `FeeManger`. As you suggested, now it is passed to `createFeeCalculatore()` as `ReadableStoreFactory`. Also `CongestionMultipliers` and `EntityUtilizationMultiplier` were refactored. Now they use `ReadableStoreFactory` instead of state.